src/index.html

```diff
@@ -11,7 +11,205 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      onsubmit="onSubmit()"
+    >
+      <fieldset class="fieldset personal-info">
+        <legend>Personal information</legend>
+
+        <div class="form-field">
+          <label for="surname">
+            Surname:
+            <input
+              type="text"
+              id="surname"
+              name="surname"
+              autocomplete="disabled"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="name">
+            Name:
+            <input
+              type="text"
+              id="name"
+              name="name"
+              autocomplete="disabled"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="age">
+            How old are You?
+            <input
+            type="number"
+            id="age"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="birth">
+            Full date of birth:
+            <input
+              type="date"
+              id="birth"
+              name="birth"
+              min="1900-01-01"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="terms">
+            I accept the term of the agreement
+            <input
+              type="checkbox"
+              id="terms"
+              name="terms"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset registration">
+        <legend>Registration</legend>
+
+        <div class="form-field">
+          <label for="email">
+            E-mail:
+            <input
+              type="email"
+              id="email"
+              name="email"
+              placeholder="email@example.com"
+              required
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="password">
+            Password:
+            <input
+              type="password"
+              id="password"
+              name="password"
+              minlength="6"
+              max="32"
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset quiz">
+        <legend>An interesting fact about you!</legend>
+
+
+        <div class="form-field">
+          Do you love cats?
```

Comment:
one blank line maximum

```suggestion
        <legend>An interesting fact about you!</legend>

        <div class="form-field">
          Do you love cats?
```